src/scrape_up/github/respository.py

```diff
@@ -148,3 +148,16 @@ def releases(self):
         else:
             message = "No releases found"
             return message
+
+    def issues_coount(self):
+
```

Comment:
Looks good @pooranjoyb, just add this method in the `documentation.md` file.

Reply:
And also resolved merge conflicts :)